src/_core/utils/TileHandler.js

```diff
@@ -134,7 +134,7 @@ export default class TileHandler {
     static _defaultKVPUrl(options) {
         let layer = options.layer;
         let url = this.mapUtil.buildTileUrl({
-            layerId: layer.get("id"),
+            layerId: layer.getIn(["wmtsOptions", "layer"]),
```

Comment:
Forget, what's the difference between these two ids?

Reply:
`layer.get("id")` is the unique id within the application, `layer.getIn(["wmtsOptions", "layer"])` is the layer id that should be used in the tile requests

Reply:
Got it. Feel like these are the same a lot of the time. Eventually we need to add comments/docs to all of our models to explain these things